### ISSUE TITLE
After using the script ./scripts/k8s/deploy_rook.sh -d to delete rook-ceph, reinstalling rook-ceph fails

### ISSUE THREAD

**Issue by dewilkin** [NVIDIA/deepops#754]
./scripts/k8s/deploy_rook.sh
+ HELM_INSTALL_DIR=/usr/local/bin
+ HELM_INSTALL_SCRIPT_URL=https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.9", GitCommit:"94f372e501c973a7fa9eb40ec9ebd2fe7ca69848", GitTreeState:"clean", BuildDate:"2020-09-16T13:56:40Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.9", GitCommit:"94f372e501c973a7fa9eb40ec9ebd2fe7ca69848", GitTreeState:"clean", BuildDate:"2020-09-16T13:47:43Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
+ . /etc/os-release
++ NAME=Ubuntu
++ VERSION='18.04.5 LTS (Bionic Beaver)'
++ ID=ubuntu
++ ID_LIKE=debian
++ PRETTY_NAME='Ubuntu 18.04.5 LTS'
++ VERSION_ID=18.04
++ HOME_URL=https://www.ubuntu.com/
++ SUPPORT_URL=https://help.ubuntu.com/
++ BUG_REPORT_URL=https://bugs.launchpad.net/ubuntu/
++ PRIVACY_POLICY_URL=https://www.ubuntu.com/legal/terms-and-policies/privacy-policy
++ VERSION_CODENAME=bionic
++ UBUNTU_CODENAME=bionic
+ case "$ID" in
+ type curl
+ type helm
+ helm version
version.BuildInfo{Version:"v3.1.2", GitCommit:"d878d4d45863e42fd5cff6743294a11d28a9abce", GitTreeState:"clean", GoVersion:"go1.13.8"}
namespace/rook-ceph created
"rook-release" has been added to your repositories
Error: rendered manifests contain a resource that already exists. Unable to continue with install: existing resource conflict: namespace: , name: 00-rook-ceph-operator, existing_kind: policy/v1beta1, Kind=PodSecurityPolicy, new_kind: policy/v1beta1, Kind=PodSecurityPolicy

**Comment by supertetelman** [comment]
I've been seeing this as well and have not found a reliable way to get past it.

It looks like if you have an existing rook deployment or if the initial deployment fails for some reason helm can get into this state. I've tried deleting the PSP resources (as well as ClusterRoles and ClusterRoleBindings) and still had issues getting Rook up.

It looks like it might be related to this Rook upgrade issue: https://github.com/rook/rook/issues/5341.

I'll keep looking into a fix, but for now I might recommend you try using the nfs-client-provisioner that I have in a new PR. We've hit a lot of issues like this with the rook deployment process so I have added an alternative storageclass.
